src/db/controller/channel.ts

```
@@ -19,15 +19,18 @@ import { RESTClient } from 'src/lib/restClient'
+import { createLoggerWithPrefix } from 'src/lib/logger'
 
 export class ChannelController {
   static tableName = 'channel_open_close'
   static channelConnectionTableName = 'channel_connection'
+  private static logger = createLoggerWithPrefix('[ChannelController] ');
   public static async feedEvents(
     rest: RESTClient,
     chainId: string,
     events: ChannelOpenCloseEvent[]
   ): Promise<() => void> {
+    this.logger.info(`feedEvents: chainId=${chainId}, events.length=${events.length}`);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)